vignettes/messydates_intro.Rmd

```diff
@@ -0,0 +1,168 @@
+---
+title: "Introduction to messydates"
+author: "Henrique Sposito"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Introduction to messydates}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
```

Comment:
This is great, @henriquesposito. Since messydates is a perhaps more targeted package than some of our others though, I wonder whether it might just make more sense to incorporate most of this into the README?